services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java

```diff
@@ -72,14 +63,27 @@ void shouldNotThrowExceptionWhenOnlyUpdateDayIsTheSame(int i) {
     // given
     long now = TimeUtils.getEpochSecondsForNow();
     long expirationDate = TimeUtils.getLastDayOfMonthForNow();
-    long lastUsedForPpa = LocalDateTime.now().minusDays(i).toEpochSecond(ZoneOffset.UTC);
+    long lastUsedForPpa = LocalDateTime.now(UTC).minusDays(i).toEpochSecond(UTC);
     ApiToken apiToken = new ApiToken("apiToken", expirationDate, now, null, lastUsedForPpa);
 
     // when - then
     assertThatNoException().isThrownBy(() -> {
       underTest.validateForPpa(apiToken);
     });
+  }
+
+  @Test
+  void shouldNotThrowExceptionWhenValidateForPpaIs23HoursSameDay() {
+    // given
+    long now = 1629762600000l; // 23:50
```

Comment:
shouldn't this be also relative to LocalDateTime.now(UTC) ? Because in validateForPpa this check is made: HOURS.between(lastUsedForPpaUtc, currentDateUtc).

Reply:
Correct! Thanks for the catch :-)